website/docs/cloud-docs/vcs/private.mdx

```diff
@@ -0,0 +1,85 @@
+---
+page_title: Connecting to Private VCS Providers - HCP Terraform
+description: >-
+  Learn how to connect HCP Terraform to private VCS providers.
+tfc_only: true
+---
+
+# Connect to Private VCS Providers
+
+This article describes the steps required to configure connect HCP Terraform to your private VCS provider using a self-hosted HCP Terraform agent.
+
+-> **Note**: Private VCS is available in limited availability. [Contact HashiCorp Sales](https://www.hashicorp.com/contact-sales) to learn more.
+
+## Background
+
+You can use HCP Terraform agents to connect HCP Terraform to your private VCS providers. The following are examples of private VCS providers that can be managed locally on your private network:
+
+- GitHub Enterprise
+- GitLab Enterprise
+- BitBucket Data Center
+
+For more information about HCP Terraform agents, refer to the [HCP Terraform agent documentation](/terraform/cloud-docs/agents).
+
+## Requirements
+
+You must meet the following requirements to configure a connection to your private VCS provider:
+
+- HCP Terraform agent v1.15.0 or newer.
+- Network connectivity between the HCP Terraform agent and private VCS provider.
+- Outbound network connectivity from the HCP Terraform agent to HCP Terraform. Refer to [networking requirements](/terraform/cloud-docs/agents/requirements#networking-requirements) for more information.
+
+## Configure the agent pool and agent
+
+Before you start your HCP Terraform agent, you must create an agent pool and token. Refer to [Create an Agent Pool](/terraform/cloud-docs/agents/agent-pools#create-an-agent-pool) for a list of steps to create an agent pool.
+
+To configure your HCP Terraform agent with private VCS support, you must set the following environment variables in the environment you run the agent. You can also set these environment variables with optional command-line flags.
+
+| Variable | Value | Command-Line Flag |
+| -------- | ----- | ----------------- |
+| `_TFC_AGENT_REQUEST_FORWARDING_ENABLED` | `true` | |
+| `TFC_AGENT_REQUEST_FORWARDING` | `true` | `-request-forwarding` |
+| `TFC_AGENT_ACCEPT` | A comma-separated list of values that  must include `ingress`. Refer to [Install and Run Agents](/terraform/cloud-docs/agents/agents#accept) for more information. | `-accept` |
+
+Below is an example command to start the HCP Terraform agent:
+
+<CodeBlockConfig hideClipboard>
+
+```bash
+$ export TFC_AGENT_TOKEN=<...>
```

Comment:
i think we should separate these commands or combine them with && the copy/paste isnt going to work here

also, can we use shell-session instead of bash?

Reply:
done!